pythnet/pythnet_sdk/src/messages.rs

```diff
@@ -30,7 +31,7 @@ use {
 /// some of the methods for PriceFeedMessage and TwapMessage are not used by the oracle
 /// for the same reason. Rust compiler doesn't include the unused methods in the contract.
 /// Once we start using the unused structs and methods, the contract size will increase.
-#[derive(Debug, Copy, Clone, PartialEq, Serialize, Deserialize)]
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
```

Comment:
Is there anyway we can keep the Copy with a variable length publisher cap list?

Reply:
I really don't like this since it means we'll have to put `.clone()` in a lot of places

Reply:
no it's not because Copy does a bit-wise copy